operators/src/source/gdal_source.rs

```diff
@@ -305,7 +315,7 @@ where
     pub fn load_tile_data(
         dataset_params: &GdalDatasetParameters,
         tile_information: &TileInformation,
-    ) -> Result<GridOrEmpty2D<T>> {
+    ) -> Result<(GridOrEmpty2D<T>, RasterProperties)> {
```

Comment:
Shouldn't we go for a wrapper struct with named fields rather than tuples? It seems to be used in more cases than just this function.

Reply:
260368a